src/SelectMulti.vue

```diff
@@ -114,7 +114,16 @@
 				open: false,
 				inputValue: ''
 			} as ComponentData
-		},
+        },
+        watch: {
+            loading(newLoadingState, oldLoadingState): void {
+                // In cases where we move from a loading state, to a resolved state...
+                if (newLoadingState === false && oldLoadingState === true) {
+                    //... we want to open the options to show the newly fetched items.
```

Comment:
Love it - nicely handled, @EvanSanderson

Reply:
Oh, wait - should we only do this if there's at least one option or a `noResultsMessage`?

Reply:
Talked about this in team discord - I think we are gonna merge and circle back on this state (ie fetch happens and no results, AND no message provided for this scenario). We may want a default state